src/fsharp/FSharp.Core/array2.fs

```diff
@@ -95,20 +93,20 @@ namespace Microsoft.FSharp.Collections
             let count2 = length2 array 
             let b1 = base1 array 
             let b2 = base2 array 
-            for i = b1 to b1+count1 - 1 do 
-              for j = b2 to b2+count2 - 1 do 
+            for i = b1 to b1 + count1 - 1 do 
+              for j = b2 to b2 + count2 - 1 do 
                 action array.[i,j]
 
         [<CompiledName("IterateIndexed")>]
         let iteri (action : int -> int -> 'T -> unit) (array:'T[,]) =
             checkNonNull "array" array
-            let count1 = length1 array 
-            let count2 = length2 array 
-            let b1 = base1 array 
-            let b2 = base2 array 
+            let count1 = length1 array
+            let count2 = length2 array
+            let b1 = base1 array
+            let b2 = base2 array
             let f = OptimizedClosures.FSharpFunc<_,_,_,_>.Adapt(action)
```

Comment:
spaces on ,

Reply:
`IDictionary<int,string>` or `IDictionary<int, string>`, I'm not sure on this one, I believe in C# code, no spaces used.

https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting there doesn't seem to be recommendations for generic type parameter.

msdn doesn't use spaces in the formatting of the doc:

https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.idictionary-2?view=netframework-4.7.2

Also looks odd if we enforce spaces after comma here but ban `IDictionary< int, string >`

Reply:
@smoothdeveloper.

Cleanup PRs are very bike-sheddable, and I should know better.  However, no C# developer would write ````var lst = new List< String >()````.   And plenty would write ````var lst = new List<string, string>()````
Many places in our codebase we use the C# convention. So I am down with following it.